testing/trino-tests/src/test/java/io/trino/security/TestAccessControl.java

```diff
@@ -184,11 +196,15 @@ public void testAccessControl()
     @Test
     public void testViewColumnAccessControl()
     {
+        TestingSystemSecurityMetadata systemSecurityMetadata = (TestingSystemSecurityMetadata) getDistributedQueryRunner().getCoordinator().getInstance(Key.get(SystemSecurityMetadata.class));
```

Comment:
maybe add a method to get `TestingSystemSecurityMetadata` to `QueryRunner`

Reply:
Are we "allowed" to add such methods for bindings which are not guaranteed to be to the expected class? I mean `SystemSecurityMetadata` is bound to `TestingSystemSecurityMetadata` in only a couple of tests.